core/src/test/java/io/questdb/test/cairo/mv/MatViewStateStoreTest.java

```diff
@@ -144,6 +144,48 @@ public void testSingleView() {
         Assert.assertEquals("table1", ordered.getQuick(1).getTableName());
     }
 
+    // loops
+    @Test
+    public void testDirectSelfLoop() {
```

Comment:
nit: these tests verify `MatViewGraph`, not `MatViewStateStore`, so maybe we should move them into `MatViewGraphTest`?

Reply:
the test suite tests both the graph and the state store. Renamed.